src/MenuItem.react.js

```diff
@@ -24,7 +24,7 @@ const BaseMenuItem = React.createClass({
           'active': active,
           'disabled': disabled,
         }, className)}>
-        <a onClick={this._handleClick} role="button">
+        <a onTouchStart={this._handleClick} onClick={this._handleClick} role="button">
```

Comment:
Fwiw, the linter is failing because the props aren't in alpha order, not because of line-length.